src/AppBundle/Resources/views/Admin/Client/Report/Formatted/checklist_formatted_body.html.twig

```diff
@@ -187,13 +187,13 @@
                     </tbody>
                 </table>
 
-                {% if report.hasSection('paDeputyExpenses') %}
-                    {% set paAppend = 'PA' %}
+                {% if report.hasSection('paDeputyExpenses') or report.hasSection('profDeputyCosts') %}
+                    {% set nonLayAppend = 'NonLay' %}
                 {% else %}
-                    {% set paAppend = '' %}
+                    {% set nonLayAppend = '' %}
                 {% endif %}
 
-                <h3 class="label question bold">{{ (page ~ '.form.accountsBalance' ~ paAppend ~ '.label') | trans({}, 'admin-checklist') }}</h3>
+                <h3 class="label question bold">{{ (page ~ '.form.accountsBalance' ~ nonLayAppend ~ '.label') | trans({}, 'admin-checklist') }}</h3>
```

Comment:
Has the NonLay label been added to a translation file somewhere? Expected it but not seeing it in the PR